include/unicore-mx/usbd/class/msc.h

```diff
@@ -64,15 +64,17 @@ typedef struct usbd_msc_backend usbd_msc_backend;
  * @param lock Lock. Optional - can be NULL
  * @param unlock Unlock. Optional - can be NULL
  */
+typedef uint32_t	msc_lba_t;
```

Comment:
Look ok to me but,

Tab between "uint32_t"	"msc_lba_t;"
(Not nitpicking) but a space is sufficient.
In other part of the code (was a typedef), you use a space instead.
Please read more code of the project and maintain the same pattern/code style.